winlogbeat/winlogbeat.yml

```diff
@@ -2,7 +2,7 @@
 
 ##################### Winlogbeat general configuration ######################
 
-#winlogbeat:
```

Comment:
Seems to be a `make update` which was missing. Is this entry even needed like this? @andrewkroh